internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,14 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	format, _ := prettyPrint([]byte(sb.String()))
+	csv.GetAnnotations()["alm-examples"] = string(format)
 	return nil
 }
+
+// prettyPrint returns a print json
+func prettyPrint(b []byte) ([]byte, error) {
```

Comment:
I prefer the name `prettifyJSON` since this function isn’t printing anything

Reply:
Done tks for the review